src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -82,14 +85,12 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
         content.add(btnClearObservers, c);
     }
 
-    private void loadCommandsFromJSON() {
-        String jsonInput = InputCommandsTextArea.getText().trim();
-        JSONCommandParser jsonCommandParser = new JSONCommandParser(jsonInput);
-        jsonCommandParser.parseToDriverCommand();
+    private void loadCommandsFromJSON(String jsonInput) {
+        InputDataModel inputDataModel = jsonCommandParser.parse(jsonInput);
```

Comment:
Just a comment: the change seems irrelevant, but now it should be easier, to actually extract this outside GUI.